src/components/SideEffects.js

```diff
@@ -41,7 +48,7 @@ const SideEffects = observer(({ assembly }) => (
                 { sr
                   .reported_symptoms
                   .map(symptom_key => assembly.translate(`survey.symptoms.${symptom_key}`))
-                  .join(", ")}
+                  .join(",")}
```

Comment:
Can we `react`-ify this a bit more, with:

```javascript
{sr
  .reported_symptoms
  .map(symptom_key => (
    <span>
      {assembly.translate(`survey.symptoms.${symptom_key}`)}
    </span>
  ))
}
```

Reply:
Ahh this helped me find a bug, we were reporting nausea rating regardless if it existed.